lib/index.js

```diff
@@ -21,10 +21,11 @@ const queueOptions = ['name', 'key', 'keys', 'exclusive', 'durable', 'autoDelete
 
 const createPublisher = (exchange, options) => {
 
-    return (message, key) => {
+    return (message, key, properties = {}) => {
 
-        Debug(`publishing to ${key || options.key}`, message);
-        exchange.publish(message, { key: key || options.key });
+        const publishOptions = Object.assign({ key: key || options.key }, properties);
```

Comment:
Shouldn't key be last?

```suggestion
        const publishOptions = Object.assign({}, properties, { key: key || options.key });
```